datasets/ar/repet/crawler.py

```diff
@@ -27,11 +27,11 @@ def parse_date(date):
     if isinstance(date, list):
         dates = []
         for d in date:
-            dates.extend(parse_date(d))
+            dates.append(parse_date(d))
         return dates
     if isinstance(date, dict):
         date = date.get("VALUE")
-    return h.parse_date(date, ["%d/%m/%Y"])
```

Comment:
this changes the return type from `List[str]` to `List[str] | str` making it harder to reason about its logic. How about returning `[date]` and keeping it `dates.extend`? Then it's a little easier to reason about the correctness of this function.